debug_module.tex

```diff
@@ -96,9 +96,11 @@ \section{Selecting Harts} \label{selectingharts}
 selects a hart, and then subsequent halt, resume, reset, and debugging
 commands are specific to that hart.
 
-A debugger can enumerate all the harts
-attached to the DM by selecting each hart starting from 0
-until \Fanynonexistent in \Rdmstatus is 1.
+To enumerate all the harts, a debugger must first determine {\tt HARTSELBITS}
```

Comment:
can we call this `HARTSELLEN` to be consistent with RISC-V `XLEN`, `ILEN`

Reply:
Yes. Done.